src/extension.ts

```diff
@@ -16,6 +17,173 @@ export function activate(context: ExtensionContext) {
 		languages.registerCodeLensProvider({ language: "typescript" }, WebpackCodeLensProvider),
 		languages.registerCodeLensProvider({ language: "typescriptreact" }, WebpackCodeLensProvider),
 
+		workspace.registerTextDocumentContentProvider("vencord-companion", {
+			async provideTextDocumentContent(uri) {
+				console.log(uri)
+				//FIXME: full uri shows up in title bar
+				const newLocal = Buffer.from(uri.path.substring(1, uri.path.lastIndexOf(".")), "base64url");
+				return newLocal.toString()
+			},
+		}),
```

Comment:
this seems like a really bad way to create a diff between two strings, but I tried for like a few hours and I couldn't figure out any other way